src/rc/start-stop-daemon.c

```diff
@@ -517,6 +526,17 @@ int main(int argc, char **argv)
 			stderr_process = optarg;
 			break;
 
+#ifdef HAVE_CAP
+		case 'B':
+			cap_iab = cap_iab_from_text(optarg);
+			if (cap_iab == NULL)
+				eerrorx("Could not parse iab: %s", strerror(errno));
+			break;
+#else
+		case 'B':
+			eerrorx("Was not compile with capability support: MKCAP=no");
```

Comment:
`MKCAP=no` is specific to the make based system.  just say generically "Capability support not enabled".